src/TurbulenceClosures/isopycnal_rotation_tensor_components.jl

```diff
@@ -47,54 +47,81 @@ rotation tensor are:
 ```
 
 where `slope² = slope_x² + slope_y²`.
+
+The slopes are tapered using the `slope_limiter.max_slope`, i.e., the tapering factor is
+`min(1, slope_limiter.max_slope² / slope²)`, where `slope² = slope_x² + slope_y²`
+that multiplies all components of the isopycnal slope tensor.
+
+References
+==========
+R. Gerdes, C. Koberle, and J. Willebrand. (1991), "The influence of numerical advection schemes
+    on the results of ocean general circulation models", Clim. Dynamics, 5 (4), 211–226.
 """
 struct SmallSlopeIsopycnalTensor{FT} <: AbstractIsopycnalTensor
     minimum_bz :: FT
 end
 
-SmallSlopeIsopycnalTensor(; minimum_bz = 0) = SmallSlopeIsopycnalTensor(minimum_bz)
+SmallSlopeIsopycnalTensor(FT::DataType=Float64; minimum_bz = FT(0)) = SmallSlopeIsopycnalTensor(minimum_bz)
 
-@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers, slope_model::SmallSlopeIsopycnalTensor) where FT
+@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid, buoyancy, tracers, slope_model::SmallSlopeIsopycnalTensor, slope_limiter)
```

Comment:
Do we want to split this into two lines, so that it's easier to read?

Reply:
Done